src/mlpack/core/distributions/laplace_distribution.hpp

```diff
@@ -64,7 +69,7 @@ class LaplaceDistribution
    * @param scale Scale of distribution.
    */
   LaplaceDistribution(const size_t dimensionality, const double scale) :
-      mean(arma::zeros<arma::vec>(dimensionality)), scale(scale) { }
+      mean(arma::zeros<VecType>(dimensionality)), scale(scale) { }
```

Comment:
```suggestion
      mean(zeros<VecType>(dimensionality)), scale(scale) { }
```
This is supposed to work as I remember